src/main/java/io/leangen/graphql/GraphQLSchemaGenerator.java

```diff
@@ -159,6 +175,7 @@ public GraphQLSchemaGenerator() {}
      * @param serviceSingletons Singletons to register as query sources
      */
     public GraphQLSchemaGenerator(Object... serviceSingletons) {
```

Comment:
Let's remove this constructor as it doesn't really serve a purpose...
Not related to the PR, but a nice opportunity to clean up the API. It _is_ a breaking change but very easy to fix plus I'm almost sure no one is using it anyway.

Reply:
Done.